app/src/main/java/com/orange/ouds/app/ui/components/button/ButtonDemoScreen.kt

```diff
@@ -53,6 +55,11 @@ fun ButtonDemoScreen() = DemoScreen(rememberButtonDemoState()) {
                 onCheckedChange = { enabled = it },
                 enabled = style == OudsButton.Style.Default
             )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_common_onColoredBackground_label),
+                checked = onColoredBox,
+                onCheckedChange = { onColoredBox = it }
+            )
```

Comment:
After today discussion, we should not allow to select negative hierarchy when onColoredBackground is enabled and vice versa.